web/src/message_list_view.ts

```diff
@@ -114,6 +115,7 @@ export type MessageGroup = {
           is_private: true;
           pm_with_url: string;
           recipient_users: RecipientRowUser[];
+          is_searched_messages: boolean;
```

Comment:
`is_search_view` would be a more natural name for this. Even better would be to negate the meaning and just say what it means, and do a `always_display_date` variable.